serverless.yml

```diff
@@ -31,7 +31,7 @@ package:
 
 custom:
   config:
-    awsAccountNumber: ${param:awsAccountNumber, '123412341234'} # SET THIS
+    awsAccountNumber: ${param:awsAccountNumber, aws:accountId}
```

Comment:
The `aws:accountId` is good, and not something I considered here. My question is if it will be 100% applicable in all cases (probably, but not sure...) or if a simple hardcoded value makes more sense as a starting point.

Also, I'd prefer having the comment `# SET THIS` as it's not for maintainers but a way to more easily find your way around as a consumer. So it's just a simple visual way to find the right place, even if crude. Please retain this as it's not "better" by removing it.